src/geometries/EdgesGeometry.js

```diff
@@ -65,13 +65,13 @@ function EdgesGeometry( geometry, thresholdAngle ) {
 
 			key = edge.toString();
 
-			if ( hash[ key ] === undefined ) {
+			if ( edges[ key ] === undefined ) {
```

Comment:
I change the name to `edges` because `hash` is for me the name of a data structure. It's like naming a variable `array` or `map`. I think it's easier to understand the code if the name of variables is related to the actual content...